docker-compose.dev.yml

```diff
@@ -126,9 +126,36 @@ services:
     ports:
       - 1025:1025 # SMTP server
       - 8025:8025 # Web UI
+  
+  openldap: # note: more advanced configuration is available
```

Comment:
Can we keep the ldap in another docker compose profile like `db-test` because the more dependencies the more time to get started with docker compose local development

Reply:
Ref: https://docs.docker.com/compose/profiles/

Reply:
Done. Now uses docker compose profiles, so the `LDAP` related services will only start up when the profile is specified in the start command; I also put a shortcut in the `Makefile` that is `make up-dev-ldap`.